cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/system/AbstractSystemMetadataWriter.java

```diff
@@ -60,15 +60,18 @@
    */
   @Override
   public void write() {
```

Comment:
is the write() method still needed?

Reply:
Yes it is still used by the artifact and dataset to write system metadata